CHANGELOG.md

```diff
@@ -1,11 +1,42 @@
 # Change Log
```

Comment:
This file is created automatically and every manual change to it is overwritten. Could you remove it from PR?